Templates/make_jenkins_EC2-Agent-Windows-instance.tmplt.json

```diff
@@ -517,9 +520,7 @@
                     "configSets": {
                         "appscript": [
                             "reboot",
-                            "make-app"
-                        ],
-                        "chainscript": [
+                            "make-app",
                             "chain-script",
```

Comment:
Presumably the `make-app` contains universal logic (i.e., installs/configurs the base agent-service and requisite "every Windows agent _needs this_" plugins) and your `chain-script` is your deployment-specific localizer. As such, the `make-app` can be conditional or not, but the `chain-script` really should be conditional (since not every deployment may need localization).